src/services/proofGenerationServices.js

```diff
@@ -23,29 +22,29 @@ export async function isBlockIncluded(blockNumber, isMainnet) {
   const rpcLength = isMainnet ? mainnetRPCLength : testnetRPCLength
   const initialRpcIndex = isMainnet ? config.mainnetRpcIndex : config.testnetRpcIndex
 
-  var matic
   var result
 
   // loop over rpcs to retry in case of an rpc error
   for (var i = 0; i < maxRetries; i++) {
     const rpcIndex = (initialRpcIndex + i) % rpcLength
     try {
       // initialize matic client
-      await initMatic(isMainnet, maticRPC[rpcIndex], ethereumRPC[rpcIndex]).then((obj) => {
-        matic = obj.matic
-      })
+      const maticClient = await initMatic(isMainnet, maticRPC[rpcIndex], ethereumRPC[rpcIndex])
 
       // check last child block included
-      const lastChildBlock = await matic.rootChain.getLastChildBlock()
+      const lastChildBlock = await maticClient.exitUtil.rootChain.getLastChildBlock()
```

Comment:
rootChain is being used multiple times, so can be stored in a variable.

Reply:
Changes done